sickbeard/providers/torrentday.py

```diff
@@ -167,27 +168,20 @@ def login(self):
                 'submit.y': 0,
             }
 
-            response = self.get_url(self.urls['login'], post_data=login_params, returns='text')
-            if not response:
+            response = self.get_url(self.urls['login'], post_data=login_params, returns='response')
+            if response.status_code not in [200]:
```

Comment:
I'm doing some more rewrites.  Replaced the tuples with dicts, and fixed some issues when you input bad formatted cookies.